### ISSUE TITLE
Error at GBitVec, stringtie version 2.0

### ISSUE THREAD

**Issue by fgypas** [gpertea/stringtie#235]
In the new version of stringtie (v2.0) with PACbio reads (mapped with minimap2) I get the following error. This is similar to #26 

```
[08/27 13:34:49]>bundle 17:3276942-3386874 [49 alignments (16 distinct), 2 junctions, 12 guides] begins processing...
[08/27 13:34:49]^bundle 17:3276942-3386874 done (1 processed potential transcripts).
[08/27 13:34:51]>bundle 17:3411370-5739308 [44909 alignments (18531 distinct), 3777 junctions, 527 guides] begins processing...
Error at GBitVec: index 10 out of bounds (size 10)
```

**Comment by fgypas** [comment]
Thank you for the quick response.

Here is the bam file for the coordinates

`samtools view results/long_reads/Hek_WT_1/minimap/aln.sorted.primary.bam -b 17:3411370-5739308 > debug.bam`

https://drive.google.com/open?id=1dsyCLQ9ik2bdKtl_yuC5fTaYTK5Slnq2

No multithreading is used. The gtf file I used is from ENSEMBL version 90. No pre-processing was performed.

Please, let me know if you need additional information.

**Comment by fgypas** [comment]
Yes, it crashes with the debug file as well. Here is the command I am using:

```
stringtie \
debug.bam \
-G Homo_sapiens.GRCh38.90.chr.gtf \
-o debug.gtf \
-A debug.tsv         \
-L \
-p 1 \
-v
```

I am using the conda installation in a 64-bit cent-os machine.

**Comment by fgypas** [comment]
I tried also with the following release but I still get the same error: https://github.com/gpertea/stringtie/releases/download/v2.0/stringtie-2.0.Linux_x86_64.tar.gz

**Comment by gpertea** [comment]
Version 2.0.1 was just released and it should fix this issue (and #227). Thank you again for sending us the data which helped with the debugging of this issue.

**Comment by fgypas** [comment]
It works. So I am closing this issue, but I opened another one here #236 with another error.
